### PR TITLE
ffe-spinner, ffe-buttons - shaky animation in IE

### PR DESCRIPTION
**Pull request by wkillerud**
## Do not merge

Opening a PR to keep track of the progress that's been made in tracking down this issue. The shaking in buttons is not fixed, and the same approach seemingly had no effect there. Also, prettier decided it liked markdown again...

#186

### CODE REVIEW

File: packages/ffe-buttons/less/base-button.less [hunk 1]
```diff
@@ -168,7 +165,7 @@
     height: 45px;
 }
 
-.ffe-button--expanded {
+.ffe-button--expanded  {
```
Comment (ivarni) [inline packages/ffe-buttons/less/base-button.less:168]: :p

**Comment by ivarni** [comment]
@wkillerud du holder fortsatt i denne ikke sant? :D

**Comment by wkillerud** [comment]
Du må gjerne ta over @ivarni 😉

**Comment by antidecaf** [comment]
Er det noen som ser på denne, eller skal vi lukke den?

**Pull request closed without merging**